### PR TITLE
Core/Unit: Holy Power bar fixed, the maximum amount of Holy Powers is…

### PR DESCRIPTION
**Pull request by ghost**
[//]: # (***************************)
[//]: # (** FILL IN THIS TEMPLATE **)
[//]: # (***************************)

Based on what @Eredar said on this issue topic [https://github.com/TrinityCore/TrinityCore/issues/18496](url), I found out that the maximum amount of holy power was 3.

**Changes proposed:**
Holy power bar should show the right amount of holy powers.
**Target branch(es):** master
- [x] master

**Tests performed:** (Does it build, tested in-game, etc.)
Tested in-game , works perfect , no crashes so far

### CODE REVIEW

File: src/server/game/Entities/Unit/Unit.cpp [hunk 1]
```diff
@@ -11132,7 +11132,8 @@ int32 Unit::GetCreatePowers(Powers power) const
 {
     if (power == POWER_MANA)
         return GetCreateMana();
-
+	if (power == POWER_HOLY_POWER)
+		return(getClass() == CLASS_PALADIN && GetUInt32Value(PLAYER_FIELD_CURRENT_SPEC_ID) == TALENT_SPEC_PALADIN_RETRIBUTION ? 5 : 0);
```
Comment (Keader) [inline src/server/game/Entities/Unit/Unit.cpp]: tabs

Comment (Skizooo) [inline src/server/game/Entities/Unit/Unit.cpp]: You are right

**Comment by Shauren** [comment]
This is a giant hack, the reason that retribution paladins currently only have 3 holy power is that SPELL_AURA_MOD_MAX_POWER is not implemented (and they do have a passive aura giving them 2 more holy power which even now is applied on them - 115675)

**Comment by Eredar** [comment]
According to wowhead:
By tooltip (8,412 – 4·20·111) on 2016/11/03 (Patch 7.1.0)		
Removed in Patch 7.0.3.

How do you know that its still there?

**Comment by Shauren** [comment]
Its listed in SkillLineAbility.db2 for retribution paladins as autolearned. Go ingame and see .list aura on yourself to confirm that its still there

**Pull request closed without merging**